src/main/java/com/example/SpringBootIntroApplication.java

```diff
@@ -0,0 +1,39 @@
+package com.example;
+
+import com.example.entity.Book;
+import com.example.service.BookService;
+import java.math.BigDecimal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringBootIntroApplication implements CommandLineRunner {
+
+    private final BookService bookService;
+
+    @Autowired
+    public SpringBootIntroApplication(BookService bookService) {
+        this.bookService = bookService;
+    }
+
+    public static void main(String[] args) {
+        SpringApplication.run(SpringBootIntroApplication.class, args);
+    }
+
+    @Override
+    public void run(String... args) throws Exception {
+        Book bookEntity = new Book();
+        bookEntity.setTitle("Book 1");
+        bookEntity.setAuthor("Author 1");
+        bookEntity.setPrice(BigDecimal.ZERO);
+        bookEntity.setDescription("sadasdad");
+        bookEntity.setIsbn("123");
+        bookEntity.setCoverImage("sd");
+        Book save = bookService.save(bookEntity);
```

Comment:
try to avoid such naming
```suggestion
        Book book = bookService.save(bookEntity);
```